deploy/manifests/dev/us-east-2/tenant/storetheindex/indexstar/canary/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/indexstar
+  - ingress.yaml
+  - pod-monitor.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
```

Comment:
The deployment is added as a patch but changes the name of the deployment. This is invalid in yaml patching; try building the canary by running `kustomize build`. I suspect it will fail.

Instead of renaming you can use `namePrefix` (or `nameSuffix`) like [this](https://github.com/ipni/storetheindex/blob/068b04ffbaa3a890642605698f856223e29d3a2b/deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/kustomization.yaml#L11-L10). You will then need to remove all the `-canary` suffixes added manually.